### ISSUE TITLE
Cannot convert enum types

### ISSUE THREAD

**Issue by jgonian** [allegro/json-avro-converter#3]
Hi, 
I'm using `tech.allegro.schema.json2avro:converter:0.2.2` and `org.apache.avro:avro:1.8.0` in order to convert some json that needs to comply to an avro schema that is using enums but the library throws an exception during the conversion. 

For example:
```
val schema = """{
               |  "type": "record",
               |  "name": "testSchema",
               |  "fields": [
               |    {
               |      "name": "eventType",
               |      "type": {
               |        "name": "EventType",
               |        "type": "enum",
               |        "symbols": ["user"]
               |      }
               |    }
               |  ]
               |}""".stripMargin

val json = """{"eventType":"user"}"""
val avro = new JsonAvroConverter().convertToAvro(json.getBytes, schema)
```
throws:
```
Not an enum: user
org.apache.avro.AvroTypeException: Not an enum: user
	at org.apache.avro.generic.GenericDatumWriter.writeEnum(GenericDatumWriter.java:164)
	at org.apache.avro.generic.GenericDatumWriter.writeWithoutConversion(GenericDatumWriter.java:106)
```

**Comment by wojtkiewicz** [comment]
I will look into this. Thanks for reporting.

**Comment by adamdubiel** [comment]
I think we fixed it with 0.2.4 - can you please check it? Works fine for us.
